Add changes to verification on update of user attribute docs. (#2496)
wso2/docs-is: en/docs/guides/identity-lifecycles/enable-verification-for-updated-mobile-number.md

        ``` 
        
        !!! note
            This publisher uses [NEXMO](https://www.nexmo.com/) as the SMS REST service provider. The 
            `api_key` and `api_secret` needs to be replaced with the appropriate values copied from the NEXMO API Dashboard. 

>             If you are need to write a custom http event publisher, see [HTTP Event Publisher](https://docs.wso2.com

Reviewer: ```suggestion
      For instructions on writing a custom http event publisher, see [HTTP Event Publisher](https://docs.wso2.com
```

1.  On the management console, navigate to **Main > Identity Providers > Resident > Other Settings > User Claim Update**.
   

> 2.  Mark **Enable user mobile number verification on update**. Additionally, you can define the expiry time (in minutes) for the verification SMS OTP to match your requirement. 

Reviewer: ```suggestion
2.  Select **Enable user mobile number verification on update**. Additionally, you can define the expiry time (in minutes) for the verification SMS OTP to match your requirement. 
```
